R/RDKit/build_tarballs.jl

```diff
@@ -1,41 +1,53 @@
 using BinaryBuilder, Pkg
 
-julia_version = v"1.6.0"
-
 name = "RDKit"
-version = v"2021.09.2"
+version = v"2022.03.1"
 
 sources = [
-    GitSource("https://github.com/rdkit/rdkit.git", "333fa5ce222627583052ddffb2c27265994a3950"),
+    GitSource("https://github.com/rdkit/rdkit.git", "7e205e0d93a3046c1eaab37120c9f6971194ddf2"),
 ]
 
 script = raw"""
 cd ${WORKSPACE}/srcdir/rdkit
+FLAGS=()
+if [[ "${target}" == *-mingw* ]]; then
+    FLAGS+=(-DRDK_BUILD_THREADSAFE_SSS=OFF -DRDK_USE_URF=OFF)
+fi
+if [[ "${proc_family}" == intel ]]; then
+    FLAGS+=(-DRDK_OPTIMIZE_POPCNT=ON)
+else
+    # For some reasons they think unconditionally enabling this option by
+    # default on all platforms is a good idea, so we have to manually disable it
+    # on non-Intel platforms
+    FLAGS+=(-DRDK_OPTIMIZE_POPCNT=OFF)
+fi
 mkdir build
 cd build
 cmake \
--DCMAKE_BUILD_TYPE=Release \
--DRDK_INSTALL_INTREE=OFF \
--DRDK_BUILD_INCHI_SUPPORT=ON \
--DRDK_BUILD_PYTHON_WRAPPERS=OFF \
--DRDK_BUILD_CFFI_LIB=ON \
--DRDK_BUILD_FREETYPE_SUPPORT=ON \
--DRDK_BUILD_CPP_TESTS=OFF \
--RDK_BUILD_SLN_SUPPORT=OFF \
--DCMAKE_INSTALL_PREFIX=${prefix} \
--DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
--DCMAKE_PREFIX_PATH=${prefix} \
-..
+    -DCMAKE_BUILD_TYPE=Release \
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} \
+    -DCMAKE_INSTALL_PREFIX=${prefix} \
```

Comment:
I must say I'm extremely confused about why we need to explicitly set `CMAKE_INSTALL_PREFIX=${prefix}`, since we already do that in the toolchain file: https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/ba6e358fee5ce57c7d4055c634edff721b322675/src/BuildToolchains.jl#L86.  But without this setting on the command line the libraries are installed in `/usr/lib`